controllers/controller.js

```diff
@@ -0,0 +1,12 @@
+const categories = require('../db/data/test-data/categories.js');
+const {selectCategories} = require('../models/model.js')
+
+
+exports.getCategories = (req, res) => {
+    selectCategories().then((Categories) => {
+      res.send({ Categories });
+    })
+    .catch((err) => {
+        next(err)
```

Comment:
Where does this error go? Currently, this .catch and next don't do anything. Consider putting in an error handler in your app that will send out a 500 Server Error or something like that if you want to - otherwise, delete these lines of code